src/index.js

```diff
@@ -67,8 +67,17 @@ async function main() {
       .trim()
 
     // Perform analysis
-    const diffChecker = new DiffChecker({ coverageReportNew, coverageReportOld, delta, changedFiles, currentDirectory, prefixFilenameUrl, prNumber });
-    
+    const diffChecker = new DiffChecker({
+      currentDirectory,
+      changedFiles,
+      coverageReportNew,
+      coverageReportOld,
```

Comment:
nitty nit: change to true lexicographical order instead of 90% sorted. 
```suggestion
      changedFiles,
      coverageReportNew,
      coverageReportOld,
      currentDirectory,
```

If you were instead optimizing for grouping related properties together, ignore this.